packages/react-emotion/package.json

```diff
@@ -19,7 +19,7 @@
     "babel-plugin-emotion": "^10.0.0-beta.2"
   },
   "peerDependencies": {
-    "react": "15.x || 16.x"
+    "react": ">= 16.3.0"
```

Comment:
This peerDep should just be removed since `@emotion/styled-base` which is a dependency of this package has react as a peerDep